src/containerapp/azext_containerapp/commands.py

```diff
@@ -92,10 +92,18 @@ def load_command_table(self, _):
         g.custom_command('create', 'create_qdrant_service', supports_no_wait=True)
         g.custom_command('delete', 'delete_qdrant_service', confirmation=True, supports_no_wait=True)
 
+    with self.command_group('containerapp service chroma', deprecate_info=self.deprecate(redirect='containerapp add-on chroma', hide=True)) as g:
```

Comment:
As we have deprecated command group `service`, we should not add `containerapp service chroma`, right?

Reply:
I'm not sure how long we're going to keep the service command up for. Let me confer with my team and get back to you on this. thank you!!

Reply:
confirmed - we'll be deprecating service (cut-off time) first release of 2024.

Reply:
is there an extension release schedule that shows version we'll be releasing first in 2024? tryna see if i can add the extra parameter to deprecation